shared/js/media/media_frame.js

```diff
@@ -371,6 +377,40 @@ MediaFrame.prototype._displayImage = function(backgroundImage, width, height) {
   var temp = this.image.clientLeft; // jshint ignore:line
 };
 
+// This function adds a label for accessibility to the image frame.
+// Videos are localized within the video player, so this is only for images.
+MediaFrame.prototype.localize = function localize() {
+  if (!this.imageblob) {
+    return;
+  }
+
```

Comment:
I think you now need to add code to handle localizing the video player if there is one.  Maybe something like this?

```
if (this.displayingVideo) {
  this.videoplayer.localize();
}
if (this.displayingImage) {
 // all the code below would go here...
}
```